source/community/release_notes/redwood.rst

```diff
@@ -25,10 +66,140 @@ Instructor Experiences
 Administrators & Operators
 **************************
 
+Settings and Toggles
```

Comment:
I generated this list of settings/toggles/flags with an edx-platform docs diff and a really naive script:


### ➕ Added
1. ANONYMOUS_SURVEY_REPORT
2. EDX_DRF_EXTENSIONS[ENABLE_JWT_AND_LMS_USER_EMAIL_MATCH]
3. ENABLE_AUTO_GENERATED_USERNAME
4. ENFORCE_SESSION_EMAIL_MATCH
5. EVENT_BUS_PRODUCER_CONFIG['org.openedx.content_authoring.course.catalog_info.changed.v1']
6. EVENT_BUS_PRODUCER_CONFIG['org.openedx.learning.course.unenrollment.completed.v1']
7. EVENT_BUS_PRODUCER_CONFIG['org.openedx.learning.xblock.skill.verified.v1']
8. FEATURES['ENABLE_BLAKE2B_HASHING']
9. FEATURES['ENABLE_CERTIFICATES_INSTRUCTOR_MANAGE]
10. FEATURES['ENABLE_COURSEWARE_SEARCH_VERIFIED_REQUIRED']
11. FEATURES['ENABLE_GRADING_METHOD_IN_PROBLEMS']
12. FEATURES['ENABLE_HIDE_FROM_TOC_UI']
13. FEATURES['ENABLE_HOME_PAGE_COURSE_API_V2']
14. FEATURES['ENABLE_LTI_PII_ACKNOWLEDGEMENT']
15. FEATURES['ENABLE_ORA_SELECTABLE_LEARNER_WAITING_REVIEW']
16. FEATURES['ENABLE_SEND_XBLOCK_LIFECYCLE_EVENTS_OVER_BUS']
17. FEATURES['SEND_LEARNING_CERTIFICATE_LIFECYCLE_EVENTS_TO_BUS']
18. JWT_AUTH_ADD_KID_HEADER:
19. SEND_TRACKING_EVENT_EMITTED_SIGNAL
20. SURVEY_REPORT_ENABLE
21. agreements.enable_lti_pii_acknowledgement
22. commerce.transition_to_coordinator.checkout
23. commerce.transition_to_coordinator.refund
24. content_tagging.disabled
25. contentstore.new_studio_mfe.use_new_certificates_page
26. contentstore.new_studio_mfe.use_new_group_configurations_page
27. contentstore.new_studio_mfe.use_new_textbooks_page
28. course_home.new_discussion_sidebar_view
29. teams.content_groups_for_teams
30. courseware.always_open_auxiliary_sidebar
31. courseware.disable_navigation_sidebar_blocks_caching
32. courseware.discovery_default_language_filter
33. courseware.enable_navigation_sidebar
34. discussions.enable_reported_content_notifications
35. enterprise.enterprise_groups_v1
36. enterprise.feature_prequery_search_suggestions
37. notifications.enable_coursewide_notifications
38. notifications.enable_email_notifications
39. notifications.enable_notifications_filters
40. notifications.enable_ora_staff_notifications
41. student.redirect_to_courseware_after_enrollment
42. studio.enable_course_update_notifications
43. user_tours.tours_disabled
44. video_config.transcript_feedback
45. video_config.xpert_translations_ui

### ❌ Removed
1. BADGR_ENABLE_NOTIFICATIONS
2. FEATURES['ENABLE_OPENBADGES']
3. FEATURES['ENABLE_SEND_XBLOCK_EVENTS_OVER_BUS']
4. SEND_CERTIFICATE_CREATED_SIGNAL
5. SEND_CERTIFICATE_REVOKED_SIGNAL
6. blockstore.use_blockstore_app_api
7. contentstore.enable_copy_paste_units
8. course_apps.proctoring_settings_modal_view
9. course_live.enable_big_blue_button
10. course_live.enable_course_live
11. courseware.learning_assistant
12. discussions.enable_learners_stats
13. discussions.enable_learners_tab_in_discussions_mfe
14. discussions.enable_moderation_reason_codes
15. discussions.enable_reported_content_email_notifications
16. learner_recommendations.enable_course_about_page_recommendations
17. learner_recommendations.enable_dashboard_recommendations
18. student.enable_2u_recommendations
19. student.enable_amplitude_recommendations
20. student.enable_fallback_recommendations

That's what I wanted to automate [in this issue](https://github.com/openedx/wg-build-test-release/issues/344), but I didn't get to it in time. So I'm adding this here just for reference, I'm not sure how accurate this list is.

Reply:
OOH thank you! Could you please link your script, *no matter how naive* to that issue? I'll work on this. Honestly this toggle business is my biggest pain point in this process and I find it a bit stressful currently.

Reply:
This sounds unbelievable, but my hard drive died a couple of weeks ago, and I lost all the work I had on there 😭

Reply:
Let me see what I can find. I'll ping you :)

Reply:
These have been integrated into the next revision. Thanks!

Reply:
Hi @feoh I see the Added flags in the doc, but I don't see all of the Removed flags. Some, but not all, appear to be mentioned in the DEPR section. Could you please review the Removed flags that María listed above, and ensure they all appear in the release notes? Thank you 🙇🏻‍♀️ 💯

Reply:
This is done in the current revision. Also noticed that my markup was all over the place in that section and should be fixed now. Let's hope ReadTheDocs agrees :)

Reply:
Wow that made - a mess. Redoing. I remain allergic to ReStructuredText apparently :(

Reply:
OK that's all fixed and all the removed toggles are there looking reasonable.

Reply:
Thanks, looks great :)